root/lib/systemd/system/titus-metatron-sync@.service

```diff
@@ -4,7 +4,7 @@ ConditionPathIsDirectory=/var/lib/titus-inits/%i/ns
 
 [Service]
 EnvironmentFile=/var/lib/titus-environments/%i.env
-ExecStart=/usr/sbin/runc --root /var/run/docker/runtime-runc/moby exec --user 0:0 --cap CAP_DAC_OVERRIDE ${TITUS_CONTAINER_ID} /titus/metatron/bin/titus-metatrond
+ExecStart=/usr/bin/runc --root /var/run/docker/runtime-${TITUS_OCI_RUNTIME}/moby exec --user 0:0 --cap CAP_DAC_OVERRIDE ${TITUS_CONTAINER_ID} /titus/metatron/bin/titus-metatrond
```

Comment:
Does this unit now break on xenial?

Reply:
It does.  Not really sure what to do about that, other than shipping a wrapper script that tries each in order.

Reply:
Or just symlink `/usr/sbin/runc` -> `/usr/bin/runc` on Bionic, I suppose.